### DISCUSSION TITLE
Kafka connector - security_protocol: does not have a value in the enumeration

### DISCUSSION THREAD

**Discussion by aleksandrpugachev** [airbytehq/airbyte#28476] [Connector Questions]
Hello everyone!
I'm trying to move data from Kafka to ClickHouse but the connector return the error - security_protocol: does not have a value in the enumeration in first step after the next exception - JSON schema validation failed. 
I use Yandex Cloud Managed Kafka (it require to use CA certificate https://cloud.yandex.com/en/docs/managed-kafka/operations/connect#get-ssl-cert)
CA certificate was installed to the trust store inside container.
My JAAS config look like
`listener.name.sasl_ssl.scram-sha-512.sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required \ username="user" \ 
password="pass";`
Full log here - [e9b6f43b_4422_4455_8a4f_95ffe2a71226_logs_1366_txt.txt](https://github.com/airbytehq/airbyte/files/12097805/e9b6f43b_4422_4455_8a4f_95ffe2a71226_logs_1366_txt.txt)
All settings from AirByte UI:
<img width="516" alt="Снимок экрана 2023-07-19 в 18 45 30" src="https://github.com/airbytehq/airbyte/assets/104512058/fee1b85c-bd2e-4825-a7ab-745f6d531c91">
<img width="529" alt="Снимок экрана 2023-07-19 в 18 45 37" src="https://github.com/airbytehq/airbyte/assets/104512058/5013d116-1805-454f-8b98-7e3231a47521">
<img width="529" alt="Снимок экрана 2023-07-19 в 18 45 11" src="https://github.com/airbytehq/airbyte/assets/104512058/8579cad1-4a2f-4d83-9b64-26ad8fe3a4e8">

**Comment by maheshreddyuma** [comment]
Same Issue brother Did You find the solution for it,

[a comment by an automated account is not shown]
